### DISCUSSION TITLE
Please add a cut out function

### DISCUSSION THREAD

**Discussion by mrieder** [flameshot-org/flameshot#3397] [Ideas]
I really like the look of Flameshot, but there is one feature that I miss and that prevents me from using the tool permanently: Cut Out.

Check out the feature in tools like Snag It and Screenpresso. Removing the middle part of a screenshot is one of the most used functions by me.
https://www.techsmith.com/learn/tutorials/snagit/cut-out/
![image](https://github.com/flameshot-org/flameshot/assets/20661789/e9c22d24-357f-4b9d-acc9-490c0a125f36)

**Comment by mmahmoudian** [comment]
There are two points that I'm fuzzy about:
1. **Use-case**: can you elaborate on your use-cases of such feature (perhaps by a toy example)
2. **Design**: Considering the UI/UX of Flameshot, how should this be implemented. I mean how does the end product look like and feel like?

**Reply by lededev** [reply]
I think this can cut off mid ad banner

**Comment by mrieder** [comment]
@mmahmoudian I'm really sorry for the late reply. I thought the reference to the other two tools was sufficient. I made a screen capture using ScreenPresso to give you a better idea. 
Use case example: I need a screenshot of the "Upload to online platforms" image on the Flameshot website, but I would like to remove the white space in the image.

https://github.com/flameshot-org/flameshot/assets/20661789/1ba4e22e-689e-4f0a-be9a-62d9edee5311

I don't know if a similar concept is feasible with Flameshot. With other tools, a screenshot is saved immediately after it is taken and then edited in a second step.
